tests/test_db.py

```diff
@@ -44,42 +44,29 @@ def get_mock_pool():
 
 @pytest.mark.asyncio
 async def test_provision_db_subsequent_run(mocker, fresh_pool):
```

Comment:
At a glance, can we drop the `mocker` class here?

Reply:
iirc this is to control order of fixtures